htlcswitch/link.go

```diff
@@ -755,6 +756,28 @@ func (l *channelLink) handleDownStreamPkt(pkt *htlcPacket, isReProcess bool) {
 		htlc.ChanID = l.ChanID()
 		htlc.ID = pkt.incomingHTLCID
 
+		// Finally, we send the HTLC message to the peer which
```

Comment:
I think this commit should be removed all together. This doesn't add anything but additional overhead. If we want to communicate directly to the peer, then we should just to so. There's no need to create this circular loop.

Reply:
Removed.